src/dashboard/components/help/HelpHome.js

```diff
@@ -24,6 +24,13 @@ export default function HelpHome(props) {
     }
   }
 
+  const RenderWebView = () => {
+    return React.createElement('webview', {
```

Comment:
Isn't this the same as `<webview src="https://docs.plottr.com" allowpopups></webview>`?
(Where the critical change is that we allowed popups.)

Reply:
If so, then I'd suggest we inline it, rather than creating a separate function for it.

Reply:
@Quiescent i cant make it work using inline like what's on the docs, i believe the custom prop is being ignored 😕

Reply:
Aha.  Ok.  Pitty :/